docs/integrations/databases/Algolia.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: "Algolia Integration in Mage"
+sidebarTitle: "Algolia Integration in Mage"
+---
+
+## Credentials Configuration
+
+Open the file named `io_config.yaml` at the root of your Mage project and enter required fields:
+
+```yaml
+version: 0.1.1
+default:
+  ALGOLIA_APP_ID: app_id
+  ALGOLIA_API_KEY: api_key
+  ALGOLIA_INDEX_NAME: index_name
+```
+
+## Required Algolia Libraries
+
+1. Add `algoliasearch==3.0.0 into `requirements.txt` to install the required weaviate library.
+2. Add `algoliasearch==3.0.0` into `setup.py` under the `all` section.
```

Comment:
The users can just add the package to their Mage `requirements.txt` file, no need to add it to tthe `setup.py` file

Reply:
Updated